api/client/run.go

```diff
@@ -121,8 +121,8 @@ func (cli *DockerCli) initSpecialVolumes(config *container.Config, hostConfig *c
 	var (
 		initConfig     *container.Config
 		initHostConfig *container.HostConfig
-		execJobs       []string
-		execID         string
+		errCh          chan error
+		execCount      uint32
```

Comment:
in the code before this patch, the execs are running parallel.
only the requests are sent in order. it is more efficient to send the requests parallel, but the changelog is confusing.